docs/introduction/faq.md

```diff
@@ -0,0 +1,109 @@
+
+## How to set up the administration with a Czech locale?
+The administration is by default in `en` locale.
+If you want to switch it to the another locale, override the method `getAdminLocale()` of the class `Shopsys\FrameworkBundle\Model\Localization\Localization`.
+This scenario is described in more detail in the tutorial [How to Set Up Domains and Locales (Languages)](./how-to-set-up-domains-and-locales.md#36-locale-in-administration).
+
+## What are the differences between "listable", "sellable", "offered" and "visible" products?
+Products can be grouped into several groups according to their current status or according to what they are used for.
+
+**Visible** - products that appear in the database as visible. 
+The conditions which the product must satisfy to appear as visible:
+- the product must not be set as hidden
+- if the attribute "selling start date" is filled in, the value of this attribute must be set to the date in the past
+- if the attribute "selling end date" is filled in, the value of this attribute must be set to the date in the future
+- the product must have a name for the specific locale
+- if the product is a variant, there must exist calculated price for this variant for the specific pricing group
+- if the product is a variant, her main variant must not be set as hidden
+- if the product is the main variant, at least one of her variants must be visible
+
+**Offered** - products that satisfied the conditions for **visible** and at the same time they appear in the database with the attribute `calculatedSellingDenied` = `FALSE`.
+The `calculatedSellingDenied` attribute shows whether the product is already sold out or if the product is a variant with the main variant that is not set up with selling denied.
+
+**Listable** - products that satisfied the conditions for **offered** and at the same time, these products are not the variants.
+By default, the specific variants are not included in the product lists.
+Only the main variants are included in the product lists.
+
+**Sellable** - products that satisfied the conditions for **offered** and at the same time, these products are not the main variants (in other words, these products are either the specific variants or they are the standard products).
+Sellable products are products that can actually be purchased.
+Only the standard products or the specific variants can actually be purchased.
+
+## How calculated attributes work?
+Some attributes that are used on the Shopsys Framework are not set directly, but their value stands out as a result of the values of other attributes.
+The values of these special attributes are calculated automatically.
+For example, if a category of products does not have a name for a locale of the specific domain, this category will be automatically set as invisible on this domain.
```

Comment:
set as not visible on this domain

it sounds better to me, because attribute name is visible

Reply:
Done